protos/bottle/fulfillment/v1/line_group.proto

```diff
@@ -0,0 +1,36 @@
+syntax = "proto3";
+
+package bottle.fulfillment.v1;
+
+import "bottle/catalog/v1/product.proto";
+
+message LineGroup {
+  string id = 1;
+  string name = 2;
+  int32 quantity = 3;
+
+  bottle.catalog.v1.Product catalog_product = 4;
```

Comment:
Pretty self explanatory, but with all these fields set, the `catalog_product` field can be null and nothing should break. I'd like to make sure we keep doing this so fulfillment and catalog don't become dependent on each other.